extensions/lifecycle/tasks/LifecycleTask.js

```diff
@@ -190,55 +205,131 @@ class LifecycleTask extends BackbeatTask {
     }
 
     /**
-     * For all incomplete MPU uploads, compare with rules (based only on prefix
-     * because no tags exist for incomplete MPU), and apply
-     * AbortIncompleteMultipartUpload rule (if any) on each upload
+     * Helper method to merge and sort Versions and DeleteMarkers lists
+     * @param {array} versions - versions list
+     * @param {array} deleteMarkers - delete markers list
+     * @return {array} merge sorted array
+     */
+    _mergeSortVersionsLists(versions, deleteMarkers) {
```

Comment:
minor: `_mergeSortedVersionsAndDeleteMarkers` could be more accurate (`sorted` instead of `sort` because we're not technically doing a merge sort, which is a special type of sort, but we're merging sorted lists together).